code/src/Components/Item.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+
+import { tasks } from '../Reducers/tasks'
+
+import { Button } from '../library/Button'
+import { Listelement } from '../Styles/Listelement'
+import { Checkbox } from './Checkbox'
+import { Text, Span } from '../Styles/Texts'
+
+import moment from 'moment'
+
+export const Item = ({ item }) => {
+  const dispatch = useDispatch()
+  const [removeBtn] = useState(true)
```

Comment:
I never seen this, what is the use of just having a variable in the useState hook? Is const a better solution since you are not changing its value on line 26? It's only for styling purposes? 
Maybe instead you can create different buttons within your lib. Example: 
```
export const RemoveButton
export const AddButton
export const ClearButton 
```
And reuse the styling that is the same. 

I seen that you are doing this in other components too.